2.6.0/spec/subscriptions.md

```diff
@@ -1,11 +1,79 @@
-# Subscriptions endpoint
+# Subscriptions Endpoint
 
 Used to fetch a list of all active subscriptions from a CO belonging to the current SP.
 
 ## Methods
 
 ### GET
 
+#### Available services for a single Access
+
+To get available services for a single access use accessId as key. The response contains details about activated services and the services
+ available for activation.
+ 
+Request
+```http
+GET /onapi/2.6/services/{accessId} HTTP/1.1
```

Comment:
I would prefer the form
```http
GET /onapi/2.6/services/?accessId={accessId} HTTP/1.1
```
I would assume an ID directly in the path would be a `subscriptionId` (which could be something we want to add later).

Reply:
There is also a mistake in the path, it should be:
`GET /onapi/2.6/subscriptions/?accessId={accessId} HTTP/1.1`

Reply:
But then we should change the get single access and get single service to the same format?
Currently it is:
Access:
GET /onapi/2.6/accesses/STTA0001 HTTP/1.1
(This should right now be "GET /onapi/2.6/accesses/{accessId} HTTP/1.1" so it needs to change no matter what)

Services:
GET /onapi/2.6/services/{accessId} HTTP/1.1